libs/datalayout/src/main/java/com/linkedin/openhouse/datalayout/datasource/TablePartitionStats.java

```diff
@@ -39,7 +39,9 @@ public PartitionStat call(Row row) {
       Row partition = row.getStruct(0);
       if (partition != null) {
         for (int i = 0; i < partition.size(); i++) {
-          values.add(partition.get(i).toString());
+          if (partition.get(i) != null) {
```

Comment:
What is the behavior we want for such null partition values? Ignore them, or handle them? Can we discuss pros and cons of either approaches.

Reply:
I changed it from ignoring to adding a 'null' value, I think this makes more sense if the point is to collect all values that are in the partition?

Reply:
@teamurko what do you think we should do here?